srcpkgs/inotify-tools/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'inotify-tools'
 pkgname=inotify-tools
-version=3.20.2.2
-revision=2
+version=3.20.11.0
+revision=1
 build_style=gnu-configure
 hostmakedepends="automake libtool"
 short_desc="Provides a simple interface to inotify"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
-homepage="https://github.com/rvoicilas/inotify-tools"
-distfiles="https://github.com/rvoicilas/${pkgname}/archive/${version}.tar.gz"
-checksum=770fb1d94ace659f975d7494e3ab8b421a6aab930b9c37c7c290ab5280abb7b8
+homepage="https://github.com/inotify-tools/inotify-tools"
+distfiles="https://github.com/inotify-tools/inotify-tools/archive/${version}.tar.gz"
+checksum=58a3cde89e4a5111a87ac16b56b06a8f885460fca0aea51b69c856ce30a37a14
 
 pre_configure() {
-	touch README
 	autoreconf -i
```

Comment:
I think using `-fi` here is best to keep it consistent between templates.

Reply:
Done